src/js/view/controls/nextuptooltip.js

```diff
@@ -164,10 +164,27 @@ export default class NextUpTooltip {
             return;
         }
 
-        // Use nextupoffset if set or default to 10 seconds from the end of playback
-        let offset = seconds(model.get('nextupoffset') || -10);
+        const configOffset = model.get('nextupoffset');
+        let offset = -10;
+        let isPercentage;
+
+        if (configOffset) {
+            isPercentage = configOffset[configOffset.length - 1] === '%';
```

Comment:
`configOffset` is not guaranteed to be a string. `(10).length` and `(10)[1]` are `undefined`.

Reply:
this would set isPercentage to undefined which would evaluate 'false' in the if.

Reply:
I believe this would evaluate to false in the isPercentage check and prevent any of the string methods from being called.

Reply:
While that is the case, given a non-string value configOffset[configOffset.length - 1] would be undefined, and isPercentage will be false, preventing any of the new logic from executing.

Reply:
You missed the most important part of the review. Use `offsetToSeconds`.